spec/models/article_spec.rb

```diff
@@ -137,3 +137,28 @@
     end
   end
 end
+
```

Comment:
I don't think this is an issue, though I understand your point. For our purposes, it doesn't make sense to compare a month with 30 days to one with 28 or 31. So this is designed to compare successive 30 day periods and is only roughly labeled month-over-month.